### PR TITLE
Added more bits to readme

### PR DESCRIPTION
**Pull request by mariaalouisaa**
Added User Stories & acceptance criteria  (which checkboxes we can tick off)
Made images smaller so they fit on a row together
Added out site link to the bottom too 😁

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,25 @@
...
+- [ ] Ensure your app is accessible to as many different users as possible.
 
-**Desktop view:**
+---
+
+### :tickets: Product Design:
+**Mobile View, desktop view & list structure:**
 
-<img src="https://user-images.githubusercontent.com/82019270/156847638-60426cdf-e0f1-4c90-8e03-45f413a7d118.png" width="60%" height="50%">
+<img src="https://user-images.githubusercontent.com/82019270/156847362-11a3dd98-8185-4a43-bf58-2b18e29ca056.png" height="300">    <img src="https://user-images.githubusercontent.com/82019270/156847638-60426cdf-e0f1-4c90-8e03-45f413a7d118.png" height="300">    <img src="https://user-images.githubusercontent.com/82019270/156847341-e50d0985-9000-429e-a48b-fb0e3a8a8869.png" height="300">
```
Comment (AsmahanM) [inline README.md:26]: Thanks for resizing the images! They were taking up quite a bit of space.

**Review by AsmahanM — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,15 +6,25 @@
...
 #### Designed and Developed by [Asmahan](https://github.com/AsmahanM), [Maria](https://github.com/mariaalouisaa) and [Peter](https://github.com/PJSalter)
 
-### :tickets: Product Design:
-**Mobile View:**
+---
+
+## 📃 User Stories & acceptence criteria ✅
 
-![Screen Shot 2022-03-04 at 21 44 04](https://user-images.githubusercontent.com/82019270/156847362-11a3dd98-8185-4a43-bf58-2b18e29ca056.png)
+- [ ] I want to be able to add tasks to a list so that I can keep track of them.
```
Comment (PJSalter) [inline README.md:13]: Thanks for adding the check boxes ✨

File: README.md (hunk 1 above)
Comment (PJSalter) [inline README.md:26]: great sizing ratio

File: README.md [hunk 3]
```diff
@@ -6,15 +6,25 @@
...
+**Mobile View, desktop view & list structure:**
 
-<img src="https://user-images.githubusercontent.com/82019270/156847638-60426cdf-e0f1-4c90-8e03-45f413a7d118.png" width="60%" height="50%">
+<img src="https://user-images.githubusercontent.com/82019270/156847362-11a3dd98-8185-4a43-bf58-2b18e29ca056.png" height="300">    <img src="https://user-images.githubusercontent.com/82019270/156847638-60426cdf-e0f1-4c90-8e03-45f413a7d118.png" height="300">    <img src="https://user-images.githubusercontent.com/82019270/156847341-e50d0985-9000-429e-a48b-fb0e3a8a8869.png" height="300">
 
-**List Structure:**
+---
 
-![Screen Shot 2022-03-04 at 21 39 02](https://user-images.githubusercontent.com/82019270/156847341-e50d0985-9000-429e-a48b-fb0e3a8a8869.png)
+🌟 Site hosted on [Github Pages](asmahanm.github.io/fac-to-do-project/) 🌟
```
Comment (PJSalter) [inline README.md:30]: 🥳 yaaaaay

**Review by PJSalter — APPROVED** [review]

**Pull request merged**